### PR TITLE
Docstrings for the commands

### PR DESCRIPTION
**Pull request by ytkimirti**

### CODE REVIEW

File: upstash_redis/client.py [hunk 1]
```diff
@@ -33,7 +52,7 @@ def __init__(
 
         self._allow_telemetry = allow_telemetry
 
-        self._rest_encoding = rest_encoding
+        self._rest_encoding: Union[Literal["base64"], None] = rest_encoding
```
Comment (mdumandag) [inline upstash_redis/client.py:55]: minor: do we need this change?

File: upstash_redis/commands.py [hunk 2]
```diff
@@ -462,6 +725,23 @@ def touch(self, *keys: str) -> ResponseT:
 
     def ttl(self, key: str) -> ResponseT:
         """
+        The seconds remaining until the key expires.
```
Comment (mdumandag) [inline upstash_redis/commands.py:728]:
```suggestion
        Returns the seconds remaining until the key expires.
```

File: upstash_redis/commands.py [hunk 3]
```diff
@@ -389,6 +557,23 @@ def pexpireat(
 
     def pttl(self, key: str) -> ResponseT:
         """
+        The milliseconds remaining until the key expires.
```
Comment (mdumandag) [inline upstash_redis/commands.py:560]:
```suggestion
        Returns the milliseconds remaining until the key expires.
```

File: upstash_redis/commands.py [hunk 4]
```diff
@@ -431,6 +660,30 @@ def scan(
         type: Union[str, None] = None,
     ) -> ResponseT:
         """
+        Return a paginated list of keys matching the pattern.
```
Comment (mdumandag) [inline upstash_redis/commands.py:663]:
```suggestion
        Returns a paginated list of keys matching the pattern.
```

File: upstash_redis/commands.py [hunk 5]
```diff
@@ -1002,6 +1468,29 @@ def hscan(
         count: Union[int, None] = None,
     ) -> ResponseT:
         """
+        Return a paginated list of keys in a hash matching the pattern.
```
Comment (mdumandag) [inline upstash_redis/commands.py:1471]:
```suggestion
        Returns a paginated list of keys in a hash matching the pattern.
```

File: upstash_redis/commands.py [hunk 6]
```diff
@@ -1110,10 +1656,17 @@ def linsert(
         self,
         key: str,
         where: Literal["BEFORE", "AFTER"],
-        pivot: str,
+        pivot: Any,
         element: str,
     ) -> ResponseT:
         """
+        Insert an element before or after another element in a list.
```
Comment (mdumandag) [inline upstash_redis/commands.py:1663]:
```suggestion
        Inserts an element before or after another element in a list.
```

File: upstash_redis/commands.py [hunk 7]
```diff
@@ -1138,6 +1693,23 @@ def lmove(
         whereto: Literal["LEFT", "RIGHT"] = "RIGHT",
     ) -> ResponseT:
         """
+        Move an element from one list to another atomically.
```
Comment (mdumandag) [inline upstash_redis/commands.py:1696]:
```suggestion
        Moves an element from one list to another atomically.
```

File: upstash_redis/commands.py [hunk 8]
```diff
@@ -1153,10 +1725,31 @@ def lmove(
 
     def lpop(self, key: str, count: Union[int, None] = None) -> ResponseT:
         """
-        See https://redis.io/commands/lpop
-        """
+        Remove and return the first element of a list.
```
Comment (mdumandag) [inline upstash_redis/commands.py:1728]:
```suggestion
        Removes and returns the first element of a list.
```

File: upstash_redis/commands.py [hunk 9]
```diff
@@ -1190,6 +1811,17 @@ def lpos(
 
     def lpush(self, key: str, *elements: str) -> ResponseT:
         """
+        Push an element to the left side of a list.
```
Comment (mdumandag) [inline upstash_redis/commands.py:1814]:
```suggestion
        Pushes an element to the left side of a list.
```

File: upstash_redis/commands.py [hunk 10]
```diff
@@ -1202,6 +1834,21 @@ def lpush(self, key: str, *elements: str) -> ResponseT:
 
     def lpushx(self, key: str, *elements: str) -> ResponseT:
         """
+        Push an element to the left side of a list, only if the list exists.
```
Comment (mdumandag) [inline upstash_redis/commands.py:1837]:
```suggestion
        Pushes an element to the left side of a list, only if the list exists.
```

File: upstash_redis/commands.py [hunk 11]
```diff
@@ -1223,6 +1889,23 @@ def lrange(self, key: str, start: int, stop: int) -> ResponseT:
 
     def lrem(self, key: str, count: int, element: str) -> ResponseT:
         """
+        Remove the first `count` occurrences of an element from a list.
```
Comment (mdumandag) [inline upstash_redis/commands.py:1892]:
```suggestion
        Removes the first `count` occurrences of an element from a list.
```

File: upstash_redis/commands.py [hunk 12]
```diff
@@ -1232,6 +1915,23 @@ def lrem(self, key: str, count: int, element: str) -> ResponseT:
 
     def lset(self, key: str, index: int, element: str) -> ResponseT:
         """
+        Set the value of an element in a list by its index.
```
Comment (mdumandag) [inline upstash_redis/commands.py:1918]:
```suggestion
        Sets the value of an element in a list by its index.
```

File: upstash_redis/commands.py [hunk 13]
```diff
@@ -1241,6 +1941,23 @@ def lset(self, key: str, index: int, element: str) -> ResponseT:
 
     def ltrim(self, key: str, start: int, stop: int) -> ResponseT:
         """
+        Trim a list to the specified range.
```
Comment (mdumandag) [inline upstash_redis/commands.py:1944]:
```suggestion
        Trims a list to the specified range.
```

File: upstash_redis/commands.py [hunk 14]
```diff
@@ -1250,6 +1967,26 @@ def ltrim(self, key: str, start: int, stop: int) -> ResponseT:
 
     def rpop(self, key: str, count: Union[int, None] = None) -> ResponseT:
         """
+        Remove and return the last element of a list.
```
Comment (mdumandag) [inline upstash_redis/commands.py:1970]:
```suggestion
        Removes and returns the last element of a list.
```

File: upstash_redis/commands.py [hunk 15]
```diff
@@ -1262,6 +1999,19 @@ def rpop(self, key: str, count: Union[int, None] = None) -> ResponseT:
 
     def rpoplpush(self, source: str, destination: str) -> ResponseT:
         """
+        Delete an element from the right side of a list and push it to the left side of another list.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2002]:
```suggestion
        Deletes an element from the right side of a list and pushes it to the left side of another list.
```

File: upstash_redis/commands.py [hunk 16]
```diff
@@ -1271,6 +2021,17 @@ def rpoplpush(self, source: str, destination: str) -> ResponseT:
 
     def rpush(self, key: str, *elements: str) -> ResponseT:
         """
+        Push an element to the right side of a list.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2024]:
```suggestion
        Pushes an element to the right side of a list.
```

File: upstash_redis/commands.py [hunk 17]
```diff
@@ -1283,6 +2044,18 @@ def rpush(self, key: str, *elements: str) -> ResponseT:
 
     def rpushx(self, key: str, *elements: Any) -> ResponseT:
         """
+        Push an element to the right side of a list only if the list exists.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2047]:
```suggestion
        Pushes an element to the right side of a list only if the list exists.
```

File: upstash_redis/commands.py [hunk 18]
```diff
@@ -1271,6 +2021,17 @@ def rpoplpush(self, source: str, destination: str) -> ResponseT:
...
         """
+        Push an element to the right side of a list.
+
+        Returns the new length of the list.
+
+        Example:
+        ```python
+        assert redis.rpush("mylist", "one", "two", "three") == 3
+
+        assert lrange("mylist", 0, -1) == ["one", "two", "three"]
```
Comment (mdumandag) [inline upstash_redis/commands.py:2032]:
```suggestion
        assert redis.lrange("mylist", 0, -1) == ["one", "two", "three"]
```

File: upstash_redis/commands.py [hunk 19]
```diff
@@ -1283,6 +2044,18 @@ def rpush(self, key: str, *elements: str) -> ResponseT:
...
+        Push an element to the right side of a list only if the list exists.
+
+        Returns the new length of the list.
+        Returns 0 if the list does not exist.
+
+        Example:
+        ```python
+        assert redis.rpushx("mylist", "one", "two", "three") == 3
+
+        assert lrange("mylist", 0, -1) == ["one", "two", "three"]
```
Comment (mdumandag) [inline upstash_redis/commands.py:2056]:
```suggestion
        assert redis.lrange("mylist", 0, -1) == ["one", "two", "three"]
```

File: upstash_redis/commands.py [hunk 20]
```diff
@@ -1295,6 +2068,15 @@ def rpushx(self, key: str, *elements: Any) -> ResponseT:
 
     def publish(self, channel: str, message: str) -> ResponseT:
         """
+        Publish a message to a channel.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2071]:
```suggestion
        Publishes a message to a channel.
```

File: upstash_redis/commands.py [hunk 21]
```diff
@@ -1309,6 +2091,32 @@ def eval(
         args: Union[List, None] = None,
     ) -> ResponseT:
         """
+        Evaluate a Lua script in the server
```
Comment (mdumandag) [inline upstash_redis/commands.py:2094]:
```suggestion
        Evaluates a Lua script in the server
```

File: upstash_redis/commands.py [hunk 22]
```diff
@@ -1330,9 +2138,23 @@ def evalsha(
         self,
         sha1: str,
         keys: Union[List[str], None] = None,
-        args: Union[List, None] = None,
+        args: Union[List[str], None] = None,
     ) -> ResponseT:
         """
+        Evaluate a Lua script in the server, cached by its SHA1 digest.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2144]:
```suggestion
        Evaluates a Lua script in the server, cached by its SHA1 digest.
```

File: upstash_redis/commands.py [hunk 23]
```diff
@@ -1467,6 +2365,12 @@ def sinterstore(self, destination: str, *keys: str) -> ResponseT:
 
     def sismember(self, key: str, member: str) -> ResponseT:
         """
+        Check if a member is in a set.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2368]:
```suggestion
        Checks if a member is in a set.
```

File: upstash_redis/commands.py [hunk 24]
```diff
@@ -1476,6 +2380,8 @@ def sismember(self, key: str, member: str) -> ResponseT:
 
     def smembers(self, key: str) -> ResponseT:
         """
+        Return all members of a set.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2383]:
```suggestion
        Returns all members of a set.
```

File: upstash_redis/commands.py [hunk 25]
```diff
@@ -1485,6 +2391,21 @@ def smembers(self, key: str) -> ResponseT:
 
     def smismember(self, key: str, *members: str) -> ResponseT:
         """
+        Check if multiple members are in a set.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2394]:
```suggestion
        Checks if multiple members are in a set.
```

File: upstash_redis/commands.py [hunk 26]
```diff
@@ -1497,6 +2418,23 @@ def smismember(self, key: str, *members: str) -> ResponseT:
 
     def smove(self, source: str, destination: str, member: str) -> ResponseT:
         """
+        Move a member from one set to another atomically.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2421]:
```suggestion
        Moves a member from one set to another atomically.
```

File: upstash_redis/commands.py [hunk 27]
```diff
@@ -1506,6 +2444,29 @@ def smove(self, source: str, destination: str, member: str) -> ResponseT:
 
     def spop(self, key: str, count: Union[int, None] = None) -> ResponseT:
         """
+        Remove and return one or more random members from a set.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2447]:
```suggestion
        Removes and returns one or more random members from a set.
```

File: upstash_redis/commands.py [hunk 28]
```diff
@@ -1518,6 +2479,29 @@ def spop(self, key: str, count: Union[int, None] = None) -> ResponseT:
 
     def srandmember(self, key: str, count: Union[int, None] = None) -> ResponseT:
         """
+        Return one or more random members from a set.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2482]:
```suggestion
        Returns one or more random members from a set.
```

File: upstash_redis/commands.py [hunk 29]
```diff
@@ -1530,6 +2514,17 @@ def srandmember(self, key: str, count: Union[int, None] = None) -> ResponseT:
 
     def srem(self, key: str, *members: str) -> ResponseT:
         """
+        Remove one or more members from a set.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2517]:
```suggestion
        Removes one or more members from a set.
```

File: upstash_redis/commands.py [hunk 30]
```diff
@@ -1548,6 +2543,29 @@ def sscan(
         count: Union[int, None] = None,
     ) -> ResponseT:
         """
+        Return a paginated list of members of a set matching a pattern.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2546]:
```suggestion
        Returns a paginated list of members of a set matching a pattern.
```

File: upstash_redis/commands.py [hunk 31]
```diff
@@ -1598,6 +2642,37 @@ def zadd(
         incr: bool = False,
     ) -> ResponseT:
         """
+        Add one or more members to a sorted set, or update its score if it already exists.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2645]:
```suggestion
        Adds one or more members to a sorted set, or updates its score if it already exists.
```

File: upstash_redis/commands.py [hunk 32]
```diff
@@ -1688,6 +2815,17 @@ def zdiffstore(self, destination: str, keys: List[str]) -> ResponseT:
 
     def zincrby(self, key: str, increment: float, member: str) -> ResponseT:
         """
+        Increment the score of a member in a sorted set.
```
Comment (mdumandag) [inline upstash_redis/commands.py:2818]:
```suggestion
        Increments the score of a member in a sorted set.
```

File: upstash_redis/commands.py [hunk 33]
```diff
@@ -1834,9 +3079,32 @@ def zrange(
         withscores: bool = False,
     ) -> ResponseT:
         """
-        See https://redis.io/commands/zrange
+        Return the members of a sorted set between a min and max value.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3082]:
```suggestion
        Returns the members of a sorted set between a min and max value.
```

File: upstash_redis/commands.py [hunk 34]
```diff
@@ -1947,6 +3282,17 @@ def zrank(self, key: str, member: str) -> ResponseT:
 
     def zrem(self, key: str, *members: str) -> ResponseT:
         """
+        Remove one or more members from a sorted set.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3285]:
```suggestion
        Removes one or more members from a sorted set.
```

File: upstash_redis/commands.py [hunk 35]
```diff
@@ -1999,6 +3351,10 @@ def zrevrange(
         self, key: str, start: int, stop: int, withscores: bool = False
     ) -> ResponseT:
         """
+        Return the members of a sorted set between a min and max value in reverse order.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3354]:
```suggestion
        Returns the members of a sorted set between a min and max value in reverse order.
```

File: upstash_redis/commands.py [hunk 36]
```diff
@@ -2075,6 +3446,32 @@ def zscan(
         count: Union[int, None] = None,
     ) -> ResponseT:
         """
+        Return a paginated list of members and their scores of an ordered set matching a pattern.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3449]:
```suggestion
        Returns a paginated list of members and their scores of an ordered set matching a pattern.
```

File: upstash_redis/commands.py [hunk 37]
```diff
@@ -2155,6 +3604,21 @@ def zunionstore(
 
     def append(self, key: str, value: str) -> ResponseT:
         """
+        Append a value to a string stored at a key.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3607]:
```suggestion
        Appends a value to a string stored at a key.
```

File: upstash_redis/commands.py [hunk 38]
```diff
@@ -2164,6 +3628,19 @@ def append(self, key: str, value: str) -> ResponseT:
 
     def decr(self, key: str) -> ResponseT:
         """
+        Decrement the integer value of a key by one.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3631]:
```suggestion
        Decrements the integer value of a key by one.
```

File: upstash_redis/commands.py [hunk 39]
```diff
@@ -2173,6 +3650,19 @@ def decr(self, key: str) -> ResponseT:
 
     def decrby(self, key: str, decrement: int) -> ResponseT:
         """
+        Decrement the integer value of a key by the given number.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3653]:
```suggestion
        Decrements the integer value of a key by the given number.
```

File: upstash_redis/commands.py [hunk 40]
```diff
@@ -2182,6 +3672,15 @@ def decrby(self, key: str, decrement: int) -> ResponseT:
 
     def get(self, key: str) -> ResponseT:
         """
+        Return the value of a key or None if the key does not exist.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3675]:
```suggestion
        Returns the value of a key or None if the key does not exist.
```

File: upstash_redis/commands.py [hunk 41]
```diff
@@ -2191,6 +3690,19 @@ def get(self, key: str) -> ResponseT:
 
     def getdel(self, key: str) -> ResponseT:
         """
+        Get the value of a key and delete the key atomically.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3693]:
```suggestion
        Gets the value of a key and delete the key atomically.
```

File: upstash_redis/commands.py [hunk 42]
```diff
@@ -2208,6 +3720,25 @@ def getex(
         persist: Union[bool, None] = None,
     ) -> ResponseT:
         """
+        Get the value of a key and optionally set its expiration.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3723]:
```suggestion
        Gets the value of a key and optionally set its expiration.
```

File: upstash_redis/commands.py [hunk 43]
```diff
@@ -2237,6 +3768,19 @@ def getex(
 
     def getrange(self, key: str, start: int, end: int) -> ResponseT:
         """
+        Return a substring of a string stored at a key.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3771]:
```suggestion
        Returns a substring of a string stored at a key.
```

File: upstash_redis/commands.py [hunk 44]
```diff
@@ -2246,6 +3790,17 @@ def getrange(self, key: str, start: int, end: int) -> ResponseT:
 
     def getset(self, key: str, value: str) -> ResponseT:
         """
+        Set the value of a key and return its old value.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3793]:
```suggestion
        Sets the value of a key and return its old value.
```

File: upstash_redis/commands.py [hunk 45]
```diff
@@ -2255,6 +3810,19 @@ def getset(self, key: str, value: str) -> ResponseT:
 
     def incr(self, key: str) -> ResponseT:
         """
+        Increment the integer value of a key by one.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3813]:
```suggestion
        Increments the integer value of a key by one.
```

File: upstash_redis/commands.py [hunk 46]
```diff
@@ -2264,6 +3832,19 @@ def incr(self, key: str) -> ResponseT:
 
     def incrby(self, key: str, increment: int) -> ResponseT:
         """
+        Increment the integer value of a key by the given number.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3835]:
```suggestion
        Increments the integer value of a key by the given number.
```

File: upstash_redis/commands.py [hunk 47]
```diff
@@ -2273,6 +3854,20 @@ def incrby(self, key: str, increment: int) -> ResponseT:
 
     def incrbyfloat(self, key: str, increment: float) -> ResponseT:
         """
+        Increment the float value of a key by the given number.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3857]:
```suggestion
        Increments the float value of a key by the given number.
```

File: upstash_redis/commands.py [hunk 48]
```diff
@@ -2282,6 +3877,19 @@ def incrbyfloat(self, key: str, increment: float) -> ResponseT:
 
     def mget(self, *keys: str) -> ResponseT:
         """
+        Return the values of all the given keys.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3880]:
```suggestion
        Returns the values of all the given keys.
```

File: upstash_redis/commands.py [hunk 49]
```diff
@@ -2292,8 +3900,20 @@ def mget(self, *keys: str) -> ResponseT:
 
         return self.execute(command)
 
-    def mset(self, values: Dict[str, str]) -> ResponseT:
+    def mset(self, values: Dict[str, Any]) -> ResponseT:
         """
+        Set multiple keys to multiple values.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3905]:
```suggestion
        Sets multiple keys to multiple values.
```

File: upstash_redis/commands.py [hunk 50]
```diff
@@ -2304,8 +3924,19 @@ def mset(self, values: Dict[str, str]) -> ResponseT:
 
         return self.execute(command)
 
-    def msetnx(self, values: Dict[str, str]) -> ResponseT:
+    def msetnx(self, values: Dict[str, Any]) -> ResponseT:
         """
+        Set multiple keys to multiple values, only if none of the keys exist.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3929]:
```suggestion
        Sets multiple keys to multiple values, only if none of the keys exist.
```

File: upstash_redis/commands.py [hunk 51]
```diff
@@ -2318,6 +3949,18 @@ def msetnx(self, values: Dict[str, str]) -> ResponseT:
 
     def psetex(self, key: str, milliseconds: int, value: str) -> ResponseT:
         """
+        Set the value of a key and set its expiration in milliseconds.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3952]:
```suggestion
        Sets the value of a key and set its expiration in milliseconds.
```

File: upstash_redis/commands.py [hunk 52]
```diff
@@ -2339,6 +3982,38 @@ def set(
         keepttl: Union[bool, None] = None,
     ) -> ResponseT:
         """
+        Set the value of a key and optionally set its expiration.
```
Comment (mdumandag) [inline upstash_redis/commands.py:3985]:
```suggestion
        Sets the value of a key and optionally set its expiration.
```

File: upstash_redis/commands.py [hunk 53]
```diff
@@ -2383,6 +4058,16 @@ def set(
 
     def setex(self, key: str, seconds: int, value: str) -> ResponseT:
         """
+        Set the value of a key and set its expiration in seconds.
```
Comment (mdumandag) [inline upstash_redis/commands.py:4061]:
```suggestion
        Sets the value of a key and set its expiration in seconds.
```

File: upstash_redis/commands.py [hunk 54]
```diff
@@ -2392,6 +4077,21 @@ def setex(self, key: str, seconds: int, value: str) -> ResponseT:
 
     def setnx(self, key: str, value: str) -> ResponseT:
         """
+        Set the value of a key, only if the key does not already exist.
```
Comment (mdumandag) [inline upstash_redis/commands.py:4080]:
```suggestion
        Sets the value of a key, only if the key does not already exist.
```

File: upstash_redis/commands.py [hunk 55]
```diff
@@ -2401,6 +4101,22 @@ def setnx(self, key: str, value: str) -> ResponseT:
 
     def setrange(self, key: str, offset: int, value: str) -> ResponseT:
         """
+        Overwrite part of a string at key starting at the specified offset.
```
Comment (mdumandag) [inline upstash_redis/commands.py:4104]:
```suggestion
        Overwrites part of a string at key starting at the specified offset.
```

File: upstash_redis/commands.py [hunk 56]
```diff
@@ -2410,6 +4126,15 @@ def setrange(self, key: str, offset: int, value: str) -> ResponseT:
 
     def strlen(self, key: str) -> ResponseT:
         """
+        Return the length of the string stored at a key.
```
Comment (mdumandag) [inline upstash_redis/commands.py:4129]:
```suggestion
        Returns the length of the string stored at a key.
```

File: upstash_redis/commands.py [hunk 57]
```diff
@@ -2419,6 +4144,27 @@ def strlen(self, key: str) -> ResponseT:
 
     def substr(self, key: str, start: int, end: int) -> ResponseT:
         """
+        Return a substring of a string stored at a key.
```
Comment (mdumandag) [inline upstash_redis/commands.py:4147]:
```suggestion
        Returns a substring of a string stored at a key.
```

File: upstash_redis/commands.py [hunk 58]
```diff
@@ -2428,6 +4174,10 @@ def substr(self, key: str, start: int, end: int) -> ResponseT:
 
     def script_exists(self, *sha1: str) -> ResponseT:
         """
+        Check if the given sha1 digests exist in the script cache.
```
Comment (mdumandag) [inline upstash_redis/commands.py:4177]:
```suggestion
        Checks if the given sha1 digests exist in the script cache.
```

File: upstash_redis/commands.py [hunk 59]
```diff
@@ -2454,6 +4208,17 @@ def script_flush(
 
     def script_load(self, script: str) -> ResponseT:
         """
+        Load the given script into the script cache
```
Comment (mdumandag) [inline upstash_redis/commands.py:4211]:
```suggestion
        Loads the given script into the script cache
```

File: upstash_redis/client.py (hunk 1 above)
Comment (ytkimirti) [reply upstash_redis/client.py:55]: For some reason literal type turns into str and it gives an error in vscode, thats why

image
image

**Review by mdumandag — APPROVED** [review]

**Pull request merged**